src/utils/crypto.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable import/prefer-default-export */
+import Sodium from 'react-native-sodium'
+import { getString } from './account'
+
+export const sign = async (message: string) => {
```

Comment:
do we still need to do any signing here in the app? Ideally all of this happens through helium-js

Reply:
This bit was copied over from the previous app. Are you thinking I can just do
```
export const makeAddGatewayTxn = async (partialTxnBin: string) => {
  const addGatewayTxn = AddGatewayV1.fromString(partialTxnBin)
  const serialized = addGatewayTxn.serialize()
  return Buffer.from(serialized).toString('base64')
}
```
in `src/utils/transactions.ts`?

Reply:
you should be able to use the `sign` function here: https://github.com/helium/helium-js/blob/master/packages/transactions/src/AddGatewayV1.ts#L118